app/core-components/navigation/NavigationBar.js

```diff
@@ -51,5 +51,5 @@ const PureNavigationBar = (): React.Node => {
 
 const NavigationBar = PureNavigationBar;
 
-export { PureNavigationBar };
+export { PureNavigationBar, PureLogo };
```

Comment:
If a sub component needs to be exported separately, it should probably be in its own file.